flink-runtime/src/test/java/org/apache/flink/runtime/security/token/KerberosDelegationTokenManagerITCase.java

```diff
@@ -175,21 +162,21 @@ void startTokensUpdate() {
                     };
 
             delegationTokenManager.startTokensUpdate();
+            ExceptionThrowingHadoopDelegationTokenProvider.throwInUsage = true;
             scheduledExecutor.triggerScheduledTasks();
             scheduler.triggerAll();
+            ExceptionThrowingHadoopDelegationTokenProvider.throwInUsage = false;
             scheduledExecutor.triggerScheduledTasks();
             scheduler.triggerAll();
             delegationTokenManager.stopTokensUpdate();
 
-            assertTrue(retryExceptionThrown.get());
             assertEquals(3, startTokensUpdateCallCount.get());
```

Comment:
Here the expectation is still 3 calls:
* Manual call in line 164
* Trigger again by exception (retry)
* Trigger again by normal re-schedule